index.html

```diff
@@ -5224,6 +5224,64 @@ <h2>IANA Considerations</h2>
         </li>
       </ul>
 
+<section id="media-type">
+  <h2>The <code>application/credential+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/credential+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Credentials format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>credential+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        Resources that use the "<code>application/credential+json</code>"
+        Media Type are required to conform to all of the requirements
+        for the "<code>application/json</code>" Media Type and are
+        therefore subject to the same encoding considerations specified
+        in Section 11 of [[!RFC7159]].
+      </td>
+    </tr>
+    <tr>
+      <td>Security considerations: </td>
+      <td>As defined in this specification.</td>
+    </tr>
+    <tr>
+      <td>Contact: </td>
+      <td>
+        W3C Verifiable Claims Working Group <a href="mailto:public-vc-wg@w3.org">public-vc-wg@w3.org</a>
+      </td>
+    </tr>
+  </table>
+
+  <p>
+    Note that while the Verifiable Credentials format uses JSON-LD
+    conventions, there are a number of constraints and additional
+    requirements for Verifiable Credential implementations that justify
+    the use of a specific media type.
+  </p>
+
+  <p>
+    Because Verifiable Credentials can be considered a restricted profile
+    of JSON-LD, Implementations SHOULD consider the
+    `application/ld+json; profile="https://www.w3.org/ns/credentials"`
+    media type as being equivalent to `application/credential+json`.
+  </p>
```

Comment:
```suggestion
```

In order to get this PR in more quickly, let's get rid of profiles for now, we can revisit profiles in a future PR.

Reply:
@msporny -- Please confirm that there are open issues for everything meant to be revisited, as it is likely that at least some will fall off the table otherwise.

Reply:
I can confirm that there are not open issues for everything meant to be revisited, given it was not clear what needed to be revisited. I do plan to raise a number of issues in the future regarding media type, but at present, those are just in my head. I'll try to find some time to write these issues down at some point later today (if I get some spare cycles).